mesh/mesh.hpp

```diff
@@ -1537,7 +1537,7 @@ class GeometricFactors
    GeometricFactors(const Mesh *mesh, const IntegrationRule &ir, int flags,
```

Comment:
Can you make `GeometricFactors(const Mesh *mesh, ...)` use `GeometricFactors(const GridFunction &nodes, ...)` to avoid code duplication?

Reply:
I added a private Init method. Let me know if that is okay, I can also have one constructor call another constructor if that is preferred.

Reply:
`Init` is really preparing the sizes to launch the `qi->Mult` - would `Eval`/`Compute`/`Mult` be more explicit?

Reply:
Sure, I'll call it compute.